controller/controller.go

```diff
@@ -542,19 +555,18 @@ func (c *Impl) handleErr(err error, key types.NamespacedName) {
 
 // GlobalResync enqueues into the slow lane all objects from the passed SharedInformer
 func (c *Impl) GlobalResync(si cache.SharedInformer) {
-	alwaysTrue := func(interface{}) bool { return true }
-	c.FilteredGlobalResync(alwaysTrue, si)
+	c.FilteredGlobalResync(c.GlobalResyncFilterFunc, si)
 }
 
 // FilteredGlobalResync enqueues all objects from the
 // SharedInformer that pass the filter function in to the slow queue.
-func (c *Impl) FilteredGlobalResync(f func(interface{}) bool, si cache.SharedInformer) {
+func (c *Impl) FilteredGlobalResync(_ func(interface{}) bool, si cache.SharedInformer) {
```

Comment:
Do we really want to ignore this parameter rather than just changing the default in GlobalResync from alwaysTrue to GlobalResyncFilterFunc (as we already do above)? Seems quite confusing to have this parameter entirely ignored, and [the readme specifically documents being able to pass a custom filter here](https://github.com/knative/pkg/blob/6f19c52bf18091ef7f344e57c72db8cbf9d95be3/injection/README.md#configstore).

Reply:
(kind of surprised that no tests failed on this change, but it looks like we currently only have tests via `GlobalResync` which is probably why :D wonder if it's worth adding some tests specifically for `FilteredGlobalResync`)

Reply:
Ack. Looks like you actually want `c.GlobalResyncFilterFunc(obj) && f(obj)` in this case.

Reply:
Nope, thanks for catching that (and the above).  I've fixed the immediate issue -- working on tests in a different commit